### PR TITLE
A review for my Trial project

### PR DESCRIPTION
**Pull request by Aseadd**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        <ul>
+            <li><a href="#email" target="_blank" class="nav-link"> Features</a></li>
+            <li><a href="#form" target="_blank" class="nav-link">How it Works</a></li>
+            <li><a href="#video" target="_blank" class="nav-link">Pricing</a></li>
+        </ul>
+    </nav>
+</header>
+
+
+    <a href="#" ></a>
```
Comment (Aseadd) [inline index.html:25]: Don't put an HTML tag without purpose

File: index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    <video id="video"  controls="controls">
+        <source src="./video/video1.mp4" type="video/mp4">
+    </video>
+    <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <input type="email" id="email" name="email" placeholder="Enter Email" />
+        <input id="submit" type="submit" value="submit" />
+    </form>
+</body>
+
```
Comment (Aseadd) [inline index.html:34]: please don't put unnecessary white spaces. the better way is to remove it

File: index.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        </ul>
+    </nav>
+</header>
+
+
+    <a href="#" ></a>
+    <video id="video"  controls="controls">
+        <source src="./video/video1.mp4" type="video/mp4">
+    </video>
+    <form id="form" action="https://www.freecodecamp.com/email-submit">
```
Comment (Aseadd) [inline index.html:29]: Don't use id for block selectors. better if you use class

File: index.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (Aseadd) [inline index.html:4]: Unnecessary white space. please try to avoid it

File: index.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Trial Project</title>
+</head>
+<body>
+<header id="header">
+    <img id="header-img" src="#" alt="Landing logo" />
+    <nav id="nav-bar">
```
Comment (Aseadd) [inline index.html:15]: Using id for block selectors containing multiple line is not advised according to HTML semantics. please use class instead

**Review by Aseadd — COMMENTED** [review]
#Change required ♻️ 

 
Please to incorporate all the comments

**Review by leonardodiasb — APPROVED** [review]
Hi @Aseadd ,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Kindly remember to add a title and a description to your Pull Requests.
- [OPTIONAL] Your comments are clear but try to be more kind and make it look more like a suggestion instead of an order.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**